src/devices/skylink_ha-434tl.c

```diff
@@ -0,0 +1,145 @@
+/* skylink off-brand wireless motion sensor and alarm system on 433.3MHz
+ * Skylink HA-434TL
+ *
+
+   This motion sensor is pretty primitive, but the price is good.  It only sends messages when it see's motion, and no motion clear message.  It also does not appear to send battery levels or low battery.  It also does not appear to send regular heartbeats as well.
+
```

Comment:
Split the lines, and to me the message every 2 hours seems like a heartbeat. The text says no heartbeat signal.